kansha/card/view.py

```diff
@@ -346,11 +206,13 @@ def render_new_card_add(self, h, comp, *args):
     return h.root
 
 
+# TODO should be in due_date extension
```

Comment:
It is required by the calendar view for now. When we have generic support for alternate board views it won't be needed anymore (hopefully).